www/src/routes/Landing.tsx

```diff
@@ -32,50 +40,67 @@ const Intro: FC = () => {
     const classes = useStyles();
 
     return (
-        <Grid container item>
-            <Grid container item style={{ height: '100vh' }}>
-                <Grid container item xs={5} alignItems='center'>
-                    <Grid item>
-                        <Box maxWidth='sm'>
-                            <img src={BlackLogo} className={classes.logo_1} />
-                        </Box>
-                        <Typography variant='h1'>CompE+</Typography>
-                        <Typography variant='h3'>Practice, Insight, and Support</Typography>
-                    </Grid>
-                </Grid>
-                <Grid container item xs={7} alignItems='center' justify='center'>
-                    <Box maxWidth='sm'>
-                        <img src={PairProgramming} className={classes.svg} />
-                    </Box>
-                </Grid>
-                <Grid container item xs={12} justify='center'>
-                    <Grid container item alignItems='center' justify='center' direction='column' spacing={3}>
+        <>
+            <Grid container item className={classes.wave_pattern} style={{ minHeight: '60vh' }}>
+                <Grid container item justify='center' spacing={5}>
+                    <Grid container item xs={4} alignItems='center'>
                         <Grid item>
-                            <Typography component='h2' variant='h2' align='center'>
-                                CompE+ is a student-led pilot program to provide support for computer engineering students looking for internships
-                            </Typography>
-                        </Grid>
-                        <Grid item>
-                            <Button variant='contained' color='primary' onClick={() => console.log('clicked Get Started button')}>
-                                {'Get Started'}
-                            </Button>
+                            <Box maxWidth='sm'>
+                                <img src={BlackLogo} className={classes.logo_1} />
+                            </Box>
+                            <Typography variant='h1'>CompE+</Typography>
```

Comment:
For limiting typography to 80 characters, you can wrap the grid items with a `<Container maxWidth='lg'>` but not the wave images themselves. I'll try to see if I can do this cleanly without nesting it toto much

Reply:
Instead of wrapping the each individual Typography, you can get away with wrapping the outermost grid that doesn't have `classes.wave_pattern`. (I also think you'll be able to get away from the negative margin issue in the previous comment.) You might need to add more spacing once you've added the containers tho